addons/website_event/views/event_templates_page_misc.xml

```diff
@@ -68,4 +37,21 @@
     </t>
 </template>
 
+<template id="page_404" name="Page Not Found">
+    <t t-call="website_event.layout">
+        <div class="o_not_editable bg-100 pt40">
+            <div class="container">
+                <div class="alert alert-info text-center d-lg-flex justify-content-between align-items-center">
+                    <p class="m-lg-0 text-info">This event page does not exist, but you can create it as you are editor of this site.</p>
+                    <a role="button" class="btn btn-info" data-post_force-top-window="true" t-attf-href="/website/add/#{path}?redirect=1#{from_template and '&amp;template=%s' % from_template}">Create Page</a>
+                </div>
+                <div class="text-center text-muted p-3">
+                    <i class="fa fa-info-circle"/> Adding the content below this line will appear on <strong>all events</strong>.
+                </div>
+            </div>
+            <hr/>
+        </div>
+    </t>
+</template>
+
```

Comment:
Aww, doesn't it handle in controller? :thinking: 
`page = request.env.user.has_group('website.group_website_designer') and 'website_event.page_404' or 'http_routing.404'`

Reply:
ha indeed, forgot about that part... I would say that it's the type of thing that should be normally handle in the template so adding a `groups="website.group_website_designer"` shouldn't hurt (just in case we want to use the template somewhere else later) :slightly_smiling_face:

Also while talking about the 404 page, it seems there is one above but seems no longer used. Perhaps I miss it, but if no longer used we could remove it

Reply:
Ah, right, the above one might left behind during rebase. I will remove it and add designer group in next push :+1: